<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wiadomosci</class>
 <widget class="QDialog" name="Wiadomosci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Załaduj tabelę</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Imie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>170</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nazwisko</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Komentarz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_comment">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Brak</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>Biblioteka &quot;KEDAR&quot;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firstname">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lastname">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_comment</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
